src/vos/vos_io.c

```diff
@@ -1379,6 +1410,9 @@ akey_update_single(daos_handle_t toh, uint32_t pm_ver, daos_size_t rsize,
 	daos_epoch_t		 epoch = ioc->ic_epr.epr_hi;
 	int			 rc;
 
+	if (vos_detect_dtx_uncertainty())
```

Comment:
definitely need a better name

Reply:
I will consider a better and clear name for that ...